README.md

```diff
@@ -1,17 +1,39 @@
 # stateless-ethereum-specs
 Specifications for the Stateless Ethereum research effort
 
+## Problem Statement and Goals
 
-## Roadmaps
+[*Problem statement and goals subject to revision and the will of all contributers to Stateless Ethereum*]
 
-* [Roadmaps for Eth1x and Stateless Ethereum](./roadmaps.md)
+The Ethereum state is ever-growing, and it will continue to do so at an accelerating pace. The growing size of the state will not only make running a full node more and more untennable for a ''normal" person, but may render the Ethereum network unstable and unusable if ignored. So we must not ignore it. 
```

Comment:
Do we have a link to a statement of the state growth problem? If not, it is OK to just leave this as-is.

Reply:
Yeah, as far as problem statements go, I don't consider this one phrased well right now. consider it a placeholder until it's talked about by more people